core/src/main/java/org/apache/calcite/tools/RelBuilder.java

```diff
@@ -1045,6 +1053,69 @@ public RelBuilder snapshot(RexNode period) {
     return this;
   }
 
+
+  /**
+   * Gets column mappings of the operator.
+   *
+   * @param op operator instance
+   * @return column mappings associated with this function
+   */
+  private Set<RelColumnMapping> getColumnMappings(SqlOperator op) {
+    SqlReturnTypeInference inference = op.getReturnTypeInference();
+    if (inference == null) {
+      return null;
+    }
```

Comment:
L1065~1067  can be removed. If it is null, the instanceof will return false.

Reply:
Ok. Will do.